README.md

```diff
@@ -45,6 +46,23 @@ WRITE_TO_CASEWORK     | Enable/Disable sending the case to iCasework (defaults t
 
 Please note if you are using IP whitelisting in the iCasework backend you will also need to add your public IP!
 
+### Environment variables for the modern-slavery-data-service <a name="env-vars-data-service"></a>
+The main modern slavery service has the ability to save an application or read a saved application.
+
+You have the option to set these additional env vars below to run the application along with the [modern-slavery-data-service](https://github.com/UKHomeOffice/modern-slavery-data-service).
+
+However if these env vars are left unset they will resolve to the defined defaults.
+
+```
```

Comment:
Are these environment variables for the data-service only? If so, it's probably not useful to put them in the main README here -- they'll inevitably diverge and get confusing.

Reply:
Good shout cheers.